<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>894</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Collect data from MQTT Subscribe Topic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>801</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Log Messages</string>
    </property>
    <widget class="QPlainTextEdit" name="pteLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>781</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbValueTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>139</y>
      <width>801</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Value Table</string>
    </property>
    <widget class="QTableWidget" name="valueTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>781</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbSubscribeTopic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>9</y>
      <width>801</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Subscribe Topic</string>
    </property>
    <widget class="QPushButton" name="btnSubscribe">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Subscribe</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>201</x>
       <y>40</y>
       <width>351</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vlQLETopicQoS">
      <item>
       <widget class="QLineEdit" name="qleTopic">
        <property name="text">
         <string>co2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbQoS"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>40</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vlLBTopicQoS">
      <item>
       <widget class="QLabel" name="lbTopic">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Topic&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbQoS">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QoS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbMinMaxAverage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>542</y>
      <width>801</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Min/Max/Average</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>107</x>
       <y>77</y>
       <width>611</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hlLEMinMaxAverage">
      <item>
       <widget class="QLineEdit" name="leMin">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leMax">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leAverage">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>601</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hlLBMinMaxAverage">
      <item>
       <widget class="QLabel" name="lbMin">
        <property name="text">
         <string>Min</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbMax">
        <property name="text">
         <string>Max</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbAverage">
        <property name="text">
         <string> Average</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionConnection"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionCreated_by_ATN"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_Excel_Data"/>
    <addaction name="actionExport_PDF_Data"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnection"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnection">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionExport_Excel_Data">
   <property name="text">
    <string>Export Excel Data</string>
   </property>
  </action>
  <action name="actionExport_PDF_Data">
   <property name="text">
    <string>Export PDF Data</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCreated_by_ATN">
   <property name="text">
    <string>Created by ATN</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
